### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by bojanpy**
Adds a very basic dockerfile, with an entrypoint that generates the ssh keys

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p /keys
+[[ ! -f  /keys/ssh_host_ed25519_key ]] && ssh-keygen -t ed25519 -f /keys/ssh_host_ed25519_key -N ""
```
Comment (donuts-are-good) [inline entrypoint.sh:4]: is that path correct or does it put it in root / keys

**Comment by bojanpy** [comment]
Yep placed the app in /app (file not folder) so root keys is where your app
looks for

**Comment by bojanpy** [comment]
It works, but obviously should be thought out a bit better

**Comment by donuts-are-good** [comment]
sounds good, lets merge now and we can modify later if you see any changes that need to be made down the road

**Pull request merged**